submissions/kasionio/oop-exercise/index.js

```diff
@@ -0,0 +1,108 @@
+/* Refer to https://github.com/OleksiyRudenko/a-tiny-JS-world for the task details
+   Complete the below for code reviewers' convenience:
+
+   Code repository: https://github.com/kasionio/a-tiny-JS-world/tree/gh-pages
+   Web app: https://kasionio.github.io/a-tiny-JS-world
+   */
+
+// ======== OBJECTS DEFINITIONS ========
+// Define your objects here
+class Inhabitant {
+  constructor(species, name, gender, legs, saying) {
+      this.species = species;
+      this.name = name;
+      this.gender = gender;
+      this.legs = legs;
+      this.saying = saying;
+      this.friends = [];
+  }
+
+  addFriends(...friendsInstances) {
+      this.friends.push(...friendsInstances);
```

Comment:
JFYI
`someArray = [...someArray, ...extraItems]` adds extra items as well

Reply:
Thanks! The spread operator looks better. 

Try to make code more ES6 looking.